src/composables/LetoModelizerApi.js

```diff
@@ -1,79 +1,18 @@
-import axios from 'axios';
```

Comment:
Rename file to UserService and put it in  `src/services` folder

Reply:
done